gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -212,6 +174,54 @@ class SetupLiveUser(object):
     def disallow_app_center_updates (self):
         self.update(GS_SCHEMA, ALLOW_UPDATES, GLib.Variant('b', False))
 
+
+def dq(s):
+    return '"{}"'.format(s)
+
+
+def remove_chrome_from_icon_grids():
+    log.info("Restoring Chromium to icon grids, if needed")
+
+    pattern = os.path.join(ICON_GRID_DIR, 'icon-grid-*.json')
+    for path in glob.glob(pattern):
+        log.info("Checking %s", path)
+        try:
+            with open(path, 'r') as f:
+                grid = f.read()
+
+            if GOOGLE_CHROME in grid:
+                log.info("Updating %s", path)
+                grid = grid.replace(dq(GOOGLE_CHROME), dq(CHROMIUM_BROWSER))
+
+                with open(path, 'w') as f:
+                    f.write(grid)
+        except:
+            log.exception("while processing %s", path)
+
+
+def prepend_installer_to_icon_grid():
+    log.info("Adding installer icon to the desktop")
+    path = os.path.join(ICON_GRID_DIR, 'icon-grid-prepend-C.json')
```

Comment:
I think there is a case where this won't quite work right...  If the image builder already specifies an icon-grid-prepend-<locale>.json, where <locale> is something other than C, that localized file will be read rather than the C, so your prepend will be ignored.